### PR TITLE
Add Missing translation

### PR DESCRIPTION
**Pull request by Evalle**

### CODE REVIEW

File: doc/ru/weechat.1.ru.adoc [hunk 1]
```diff
@@ -22,10 +22,9 @@ weechat - расширяемый чат-клиент
 
 == ОПИСАНИЕ
 
-// TRANSLATION MISSING
-WeeChat (Wee Enhanced Environment for Chat) is a free chat client, fast and
-light, designed for many operating systems.
-It is highly customizable and extensible with scripts.
+WeChat (Web Enhanced Окружающая среда для чат) это свободным чат-клиент, быстрый и легкий,
```
Comment (Zarthus) [inline doc/ru/weechat.1.ru.adoc]: You missed an `e` here, it's `WeeChat`, not `WeChat`

**Comment by Evalle** [comment]
Thanks guys, I missed it somehow, gonna fix it in a couple of minutes.

**Comment by Evalle** [comment]
Fixed @flashcode @Zarthus

**Comment by flashcode** [comment]
Merged.

**Pull request closed without merging**